store/sqlstore/command_store.go

```diff
@@ -64,7 +66,14 @@ func (s SqlCommandStore) Save(command *model.Command) (*model.Command, *model.Ap
 func (s SqlCommandStore) Get(id string) (*model.Command, *model.AppError) {
 	var command model.Command
 
-	if err := s.GetReplica().SelectOne(&command, "SELECT * FROM Commands WHERE Id = :Id AND DeleteAt = 0", map[string]interface{}{"Id": id}); err != nil {
+	sql, args, err := s.getQueryBuilder().
+		Select("*").
+		From("Commands").
+		Where(sq.Eq{"Id": id, "DeleteAt": 0}).ToSql()
+	if err != nil {
+		return nil, model.NewAppError("SqlCommandStore.Get", "store.sql_command.save.get.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
For consistency, errors when building queries should use the `store.sql.build_query.app_error` message:

```suggestion
		return nil, model.NewAppError("SqlCommandStore.Get", "store.sql.build_query.app_error", nil, err.Error(), http.StatusInternalServerError)
```